src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -252,14 +254,23 @@ export class DynamicFacet extends Component implements IAutoLayoutAdjustableInsi
      * **Note:**
      * > The [`DynamicFacetRange`]{@link DynamicFacetRange} component does not support this option.
      */
-    valueCaption: ComponentOptions.buildJsonOption<IStringMap<string>>()
+    valueCaption: ComponentOptions.buildJsonOption<IStringMap<string>>(),
+
+    /**
+     * The id of another facet in which at least one value must be selected in order
```

Comment:
Is it possible to make a dynamic facet depend on a non-dynamic facet (or category facet) and vice-versa?

If so, the current `dependsOn` comment is accurate, and my documentation review ends here :)

Reply:
Yes it should be possible!